n2y/main.py

```diff
@@ -61,67 +65,83 @@ def name_column_valid(raw_rows, name_column):
 
     # make sure title column is not empty (only the first row is checked)
     if first_row_flattened[name_column] is None:
-        print(f"Column \"{name_column}\" cannot be empty.", file=sys.stderr)
-        return False
+        raise NotImplementedError(f"ERROR: Column \"{name_column}\" Cannot Be Empty.")
 
     # make sure the title column is a string
     if not isinstance(first_row_flattened[name_column], str):
-        print(f"Column \"{name_column}\" does not contain a string.", file=sys.stderr)
-        print("Available column(s): ",
-              # only show columns that have strings as possible options
-              ", ".join(available_columns()), file=sys.stderr)
-        return False
+        raise NotImplementedError("%s%s" % (
+            f"ERROR: Column \"{name_column}\" Does Not Contain A String.\n",
+            # only show columns that have strings as possible options
+            "Available Column(s): " + ", ".join(available_columns())))
     return True
 
 
 def export_markdown(client, raw_rows, options):
     file_names = []
+    skips = {'Empty': 0, 'Unnamed': 0, 'Duplicate': 0}
     for row in raw_rows:
         meta = simplify.flatten_database_row(row)
         page_name = meta[options.name_column]
-        filename = re.sub(r"[/,\\]", '_', page_name.lower())
         if page_name is not None:
-            print(f"Processing {meta[options.name_column]}", file=sys.stderr)
-            if filename in file_names:
-                print(f'WARNING: duplicate file name "{filename}.md"', file=sys.stderr)
-            file_names.append(filename)
-
-            pandoc_output = converter.load_block(client, row['id']).to_pandoc()
-            # do not create markdown pages if there is no page in Notion
-            if pandoc_output:
-                markdown = pandoc.write(pandoc_output, format='gfm+tex_math_dollars') \
-                    .replace('\r\n', '\n')  # Deal with Windows line endings
-
-                # create target path if it doesn't exist
-                # if not os.path.exists(options.target):
-                os.makedirs(options.target, exist_ok=True)
-
-                # sanitize file name just a bit
-                # maybe use python-slugify in the future?
-                with open(os.path.join(options.target, f"{filename}.md"), 'w') as f:
-                    f.write('---\n')
-                    f.write(yaml.dump(meta))
-                    f.write('---\n\n')
-                    f.write(markdown)
+            filename = re.sub(r"[\s/,\\]", '_', page_name.lower())
+            if filename not in file_names:
+                file_names.append(filename)
+
+                pandoc_output = converter.load_block(client, row['id']).to_pandoc()
+                # do not create markdown pages if there is no page in Notion
+                if pandoc_output:
+                    logger.warning(f'Processing Page "{meta[options.name_column]}"')
```

Comment:
```suggestion
                    logger.info('Processing Page "%s"', meta[options.name_column])
```

This is not a `warning`. Should be `info` or `debug`. Also, the logger already has a way of composing strings. See https://docs.python.org/3/howto/logging.html#loggers